.circleci/config.yml

```diff
@@ -528,6 +607,26 @@ jobs:
           registry: ${DOCKER_PRIMARY_REGISTRY}
           secondaryregistry: ${DOCKER_SECONDARY_REGISTRY}
 
+  devpi_synapse_3xx:
+    docker:
+      - image: cimg/python:3.11
+    steps:
+      - checkout
+      - run:
+          name: Version bump and date the build.
+          command: |
+            python3 -m pip install bump2version
+            DATE=`date '+%Y%m%d%H%M'`
+            CURRENT_VERSION=`grep "current_version =" .bumpversion.cfg | cut -f 2 -d "=" | tr -d '[:space:]'`
```

Comment:
I think this also needs to strip off the previous `a{DATE}` portion of the version

Reply:
We're not persisting these values in git, so we shouldn't need to strip anything here. They are only being used to publish the wheels to the internal index.

Reply:
Oh I see, gotcha